src/components/FeatureViews/Search/Search.js

```diff
@@ -272,7 +272,7 @@ const Search = (props) => {
                             <FormControl required variant="outlined" className={"search-selector"}>
                                 <Select className={'select-primary-filter'}
                                     // classNamePrefix={''}
-                                    disableUnderline
+                                    disableunderline="true"
```

Comment:
refactor this to just `disableunderline` you don't need it to be `disableunderline="true"`

React will treat this as the same

Reply:
String `"true"` is different from boolean `true`.